### PR TITLE
add detail page

### PR DESCRIPTION
**Pull request by Sota-hub**

### CODE REVIEW

File: navigation/HomeNavigator.js [hunk 1]
```diff
@@ -1,26 +1,70 @@
...
 import CustomHeaderButton from "../components/common/CustomHeaderButton";
-import HomeScreen from "../screens/HomeScreen";
+import PostList from "../screens/PostList";
+import PostDetail from "../screens/PostDetail";
+import PostCreate from "../screens/PostCreate";
 import defaultNavOptions from "./defaultNavOptions";
+import Colors from "../constants/Colors";
+import { FontAwesome } from "@expo/vector-icons";
+
+const leftHeader = (navigation) => (
```
Comment (SOunit) [inline navigation/HomeNavigator.js:12]: variable name should be menuButton or something.
leftHeader is a name for position. content and name should be match.

File: screens/PostCreate.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import { View, StyleSheet } from "react-native";
+import { Input, Button, Card } from "react-native-elements";
+
+const PostCreate = () => {
```
Comment (SOunit) [inline screens/PostCreate.js:4]: maybe this should be a commentCreate?

ref
https://slack-imgs.com/?c=1&o1=ro&url=https%3A%2F%2Fi.stack.imgur.com%2FAyIkW.png

Comment (SOunit) [inline screens/PostCreate.js:4]: maybe this should be a CommentCreate?

ref
https://slack-imgs.com/?c=1&o1=ro&url=https%3A%2F%2Fi.stack.imgur.com%2FAyIkW.png

File: screens/PostDetail.js [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+import { Feather } from "@expo/vector-icons";
+import { getDatabase, ref, update } from "firebase/database";
+import Colors from "../constants/Colors";
+
+const PostDetail = ({ navigation }) => {
+  const posts = useSelector((state) => state.posts); // [{},{}]
+  const postId = navigation.getParam("id");
+  const userId = useSelector((state) => state.auth.userId);
+
+  // find chosen question
```
Comment (SOunit) [inline screens/PostDetail.js:13]: question -> post

File: store/reducers/posts.js [hunk 4]
```diff
@@ -0,0 +1,12 @@
+import { STORE_POSTS } from "../actions/posts";
+
+const initialState = [];
+
+export default (state = initialState, action) => {
+  switch (action.type) {
+    case STORE_POSTS:
+      return action.payload;
```
Comment (SOunit) [inline store/reducers/posts.js:8]: overwrite?
not concat?
return {...state, action.payload}

**Review by SOunit — CHANGES_REQUESTED** [review]
i added some comments. pls check them!

**Comment by Sota-hub** [comment]
I fixed your indications

File: store/reducers/posts.js (hunk 4 above)
Comment (SOunit) [reply store/reducers/posts.js:8]: no old state, because this is init data

**Review by SOunit — APPROVED** [review]
lgtm

**Pull request merged**